tools/libboard/board_n5010/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+## Copyright(c) 2019-2020, Intel Corporation
```

Comment:
Please see note pertaining to dual-copyright files.

Reply:
Fixed